### DISCUSSION TITLE
How do I check if the bot is already playing with slash commands?

### DISCUSSION THREAD

**Discussion by SandSeppel** [Rapptz/discord.py#9901] [Q&A] [answered]
I know there is a voice client class with the is_playing function but it don't know how to navigate to it from interaction.

**Comment by lmaotrigine** [comment, marked as answer]
You can go through [Interaction.guild](https://discordpy.readthedocs.io/en/stable/interactions/api.html#discord.Interaction.guild) and [Guild.voice_client](https://discordpy.readthedocs.io/en/stable/api.html#discord.Guild.voice_client).
